core/base-service/legacy-request-handler.spec.js

```diff
@@ -185,7 +185,7 @@ describe('The request handler', function () {
         )
 
         const { headers } = await got(`${baseUrl}/testing/123.json`)
-        expect(headers['cache-control']).to.equal('max-age=300, s-maxage=300')
+        expect(headers['cache-control']).to.equal('max-age=200, s-maxage=200')
```

Comment:
this made me do a double take on first pass, and I think it's because I was reading the `createFakeHandlerWithCacheLength` function name a _little_ too literally resulting in me thinking this showed that a user could set a _lower_ value with `cacheLength` (i.e. I think I just misinterpreted what the `200` up on line 177 was being used for